### DISCUSSION TITLE
Sep_div_002: dati obbligatori su versione 1.26

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#770] [Supporto servizi cooperativi] [answered]
Con la versione 1.26, per il caso d'uso in oggetto

Sep_Div_002 - 342000 - Separazioni e divorzi: Accordo di scioglimento o di cessazione degli effetti civili del matrimonio divorzio

si introduce - come obbligatoria - la sezione:

![image](https://github.com/italia/ansc/assets/26415430/80b6b1ab-9d1a-42e6-ad0d-5dbc382b3bbb)

nel caso in cui "TipoProvvedimentoSeparazione" assume valore "3".
Si tratta del caso in cui i due coniugi, prima di sottoscrivere l'accordo per la cessazione (ovvero lo scioglimento) del matrimonio, si fossero separati legalmente mediante accordo dinanzi all'usc. La formula di riferimento l'accordo di cessazione (ovvero scioglimento) 
del matrimonio dinanzi all'usc è la 121-quater all'interno della quale, però, in riferimento alla separazione dei due coniugi, è previsto la sola indicazione dell' _accordo di separazione_ (nessuna indicazione, invece, è richiesta in riferimento alla _conferma dell'accordo di separazione_):

![image](https://github.com/italia/ansc/assets/26415430/89a75afe-9115-4b89-8041-04d26c2fcc39)

Chiediamo pertanto che sia rimossa l'indicazione di obbligatorietà per la sezione di cui sopra.

Rimaniamo in attesa di vostra indicazione

**Comment by mttfranci** [comment]
Salve, stiamo verificando la vostra segnalazione.

Saluti

Assistenza ANSC

**Comment by asqueglia** [comment, marked as answer]
Buongiorno @antonioluberto, dalle verifiche effettuate abbiamo appurato che la sezione è stata aggiunta a seguito di una esplicita richiesta da parte del comune di Lissone, issue GH #709  confermata da ANUSCA.
